04-vector-raster-integration-advanced.Rmd

```diff
@@ -440,33 +470,34 @@ head(av_tree_height_tower)
 #how many pixels were extracted
 nrow(av_tree_height_tower)
 
-
 ```
 
 Notice that the command above extracted 1259 pixels from our LiDAR derived canopy
```

Comment:
Just delete this text.
` Notice that the command above extracted 1259 pixels from our LiDAR derived canopy`
 i was testing having them first extract all pixels and then summarize. not sure how it will be taught!